pkgs/os-specific/linux/numad/default.nix

```
@@ -1,26 +1,25 @@
-{ lib, stdenv, fetchgit }:
+{ lib, stdenv, fetchgit, unstableGitUpdater, writeShellScript }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "numad";
-  version = "0.5";
+  version = "0.5-unstable-2023-09-06";
 
   src = fetchgit {
     url = "https://pagure.io/numad.git";
-    rev = "334278ff3d774d105939743436d7378a189e8693";
-    sha256 = "sha256-6nrbfooUI1ufJhsPf68li5584oKQcznXQlxfpStuX5I=";
+    rev = "3399d89305b6560e27e70aff4ad9fb403dedf947";
+    sha256 = "sha256-USEffVcakaAbilqijJmpro92ujvxbglcXxyBlntMxaI=";
   };
 
-  hardeningDisable = [ "format" ];
-
-  patches = [
-    ./numad-linker-flags.patch
-  ];
   postPatch = ''
     substituteInPlace Makefile --replace "install -m" "install -Dm"
   '';
 
   makeFlags = [ "prefix=$(out)" ];
 
+  passthru.updateScript = unstableGitUpdater {
+    tagConverter = writeShellScript "tagConverter" ''( read tag; test "$tag" = "0" && tag=0.5; echo "$tag" )'';
```

Well into nit-territory now, but we can save ourselves a sub-shell
```suggestion
    tagConverter = writeShellScript "tagConverter" ''
      read tag; test "$tag" = "0" && tag=0.5; echo "$tag"
    '';
```

---

Should be fixed now